### PR TITLE
Fix embedded views in other fields

### PR DESCRIPTION
**Pull request by omarkasem**
- Fixes https://github.com/GravityKit/GravityView/issues/2090
- The wpautop function breaks the scripts from Datatables, So I disabled them for all gv views fields in this area

💾 [Build file](https://www.dropbox.com/scl/fi/iwvs16imny6ao0sp0apf2/gravityview-2.28.0-03f043413.zip?rlkey=a1k6wa56fxkovhjvoicpxebw3&dl=1) (03f043413).

### CODE REVIEW

File: includes/class-api.php [hunk 1]
```diff
@@ -1627,7 +1627,7 @@ function gravityview_field_output( $passed_args, $context = null ) {
 		return '';
 	}
 
-	if ( '' !== $placeholders['value'] && ! empty( $args['wpautop'] ) ) {
+	if ( '' !== $placeholders['value'] && ! empty( $args['wpautop'] ) && (isset($field['id']) && $field['id'] !== 'gravityview_view') ) {
```
Comment (zackkatz) [inline includes/class-api.php:1630]: Thanks for this, @omarkasem.

Please:

1. Fix spacing
2. Assign to a variable outside the logic check
3. Use `\GV\Utils::get( $field, 'id' )` intead of checking isset

**Review by zackkatz — CHANGES_REQUESTED** [review]

File: includes/class-api.php (hunk 1 above)
Comment (omarkasem) [reply includes/class-api.php]: @zackkatz 
All done 
But I think @mrcasual Told me before that we want to phase out the use of the legacy Utils helper class, So should i try to use it more or not?

Comment (mrcasual) [reply includes/class-api.php]: @omarkasem, correct, I asked not to rely less on helpers and instead use native PHP. 

`(isset($field['id']) && $field['id'] !== 'gravityview_view'` 

can be expressed as 

`'gravityview_view' !== ( $field['id'] ?? '' )`.

**Comment by mrcasual** [comment]
@Mwalek, please test.

**Comment by Mwalek** [comment]
@mrcasual this is working as expected.

E.g: https://basecamp.try.gravitykit.com/view/issue-2090-view/entry/26/

**Pull request merged**